### ISSUE TITLE
显存占用明显增加

### ISSUE THREAD

**Issue by tricksterANDthug** [PaddlePaddle/Paddle#42198]
### 请提出你的问题 Please ask your question

for epoch in range(epoch_num):
        print(f'Epoch {epoch + 1} / {epoch_num}')
        model.train()
        for batch_id, data in enumerate(train_loader()):
            loss, acc, miou = train(data, model)
            # save log
            writer.add_scalar('train/loss', step=train_step, value=loss.numpy()[0])
            writer.add_scalar('train/acc', step=train_step, value=acc)
            writer.add_scalar('train/miou', step=train_step, value=float(miou))
            train_step += 1
            # print loss and acc
            print({'loss': loss.numpy()[0], 'acc': acc, 'miou': miou})
            loss.backward()
            optimizer.step()
            optimizer.clear_grad()

        model.eval()
        for batch_id, data in enumerate(val_loader()):
            loss, acc, miou = train(data, model)
            writer.add_scalar('val/loss', step=val_step, value=loss.numpy()[0])
            writer.add_scalar('val/acc', step=val_step, value=acc)
            writer.add_scalar('val/miou', step=val_step, value=float(miou))
            val_step += 1
            # print loss and acc
            print({'loss': loss.numpy()[0], 'acc': acc, 'miou': miou})

        # checkpoint_path = r'./checkpoints/'
        # if not os.path.exists(checkpoint_path):
        #     os.mkdir(checkpoint_path)
        # model.save(os.path.join(checkpoint_path, 'model'))

这是我的训练的代码，当代码运行到model.train()的时候，显存占用是8.8g，当运行到model.eval()的时候，显存直接占到15.8g，请问这个现象正常吗
